internal/fs/inode/file.go

```diff
@@ -673,6 +696,15 @@ func (f *FileInode) CacheEnsureContent(ctx context.Context) (err error) {
 }
 
 func (f *FileInode) CreateEmptyTempFile() (err error) {
```

Comment:
Shall we rename this function to `CreateWriteHandler()` or `CreateBufferedOrTempWriter()`